onnxruntime/test/contrib_ops/beam_search_test.cc

```diff
@@ -394,5 +396,15 @@ TEST(BeamSearchTest, GptBeamSearchFp16_VocabPadded) {
   }
 }
 
+TEST(BeamSearchTest, T5WithSequenceInputIds) {
+  ModelTester tester(CurrentTestName(), ORT_TSTR("testdata/tiny_t5_with_sequence_input_ids.onnx"));
+  tester.AddInput("encoder_input_ids", {1, 5}, {16, 17, 1, 0, 8});
+  tester.AddOutput("sequences", {1, 3, 10}, {2, 19, 18, 3, 8, 8, 8, 8, 8, 8, 2, 19, 18, 3, 10, 19, 18, 3, 8, 8, 2, 19, 18, 15, 13, 13, 13, 13, 13, 13});
+#ifdef USE_CUDA
+  tester.ConfigEp(DefaultCudaExecutionProvider());
+#endif
+  tester.RunWithConfig();
```

Comment:
This might run in other EP (like CPU and DML etc).

I think there is no implementation for CPU EP for this feature.

Reply:
you mean the `sequence_as_input_ids`? we used that for months in both CPU and CUDA (with this fix) EPs, maybe only test coverage was missing

Reply:
ok, I see now what you mean. This is also why the test is failing on other pipelines as well. I pushed a commit to make sure test is restricted on CPU and CUDA EPs